packages/mui-codemod/codemod.js

```diff
@@ -162,6 +162,11 @@ function run(argv) {
 
   runJscodeshiftTransform(codemod, files, flags, argv._);
   runPostcssTransform(codemod, files);
+  if (!flags.skipPrettier) {
+    childProcess.spawnSync('prettier', ['--write', `prettier --write $(git diff --name-only)`], {
```

Comment:
Feels a bit heavy to spawn an extra process, wouldn't it be possible as a jscodeshift transformer?

```tsx
import * as prettier from 'prettier';

export default async function transformer(file) {
  const prettierConfig = await prettier.resolveConfig(file.path);
  return prettier.format(file.source, prettierConfig);
}
```

Almost feels like those APIs were made for each other 😄

Reply:
1. We need to add prettier as a dependency, so using `npx` will be a bit slower, right?
2. Adding this requires a lot of changes to the expected files. I try to avoid it.

Reply:
1. I suppose there could be a one time cost in some cases, looks like prettier total install size is about 50% of jscodeshift total install size (total size = package + dependencies). I'm not sure npx install size is something we should optimize for here, ideally this will be run at most once ever over the lifetime of a project.
2. I suppose you can still make it optional with a `skipPrettier` flag

Reply:
> 1. ideally this will be run at most once ever over the lifetime of a project

If the codemod release a new version, isn't another installation is required?

> I suppose you can still make it optional with a skipPrettier flag

Yes, it can be skipped.

Reply:
I tried timing the installation of `jscodeshift` alone and also `jscodeshift` and `prettier` together and the difference seems to be insignificant (<10ms)